src/libopenrave/planner.cpp

```diff
@@ -180,6 +174,18 @@ PlannerStatus& PlannerStatus::SetPlannerParameters(PlannerParametersConstPtr par
     return *this;
 }
 
+// This assumes that collisionReport plink1 and plink2 fields are populated! Current implementation does not check if vLinkColliding is populated (when is it populated)?
+void PlannerStatus::UpdatePlannerStatusInfo(CollisionReport& collisionReport)
```

Comment:
I don't think this name conveys much useful information. Consider a better name that reflects what it does.